diagnostic.md

```diff
@@ -5,74 +5,87 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+The backend is for updating, creating, reading, and deleting data.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+The Model
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+The Controller
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+Great question. Its the way of the dianosour some might say.
```

Comment:
Right but we don't use them on the back-end because we handle them on the front-end!